symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/SlashCommand.java

```diff
@@ -105,11 +107,29 @@ protected ActivityInfo info() {
     return new ActivityInfo()
         .type(ActivityType.COMMAND)
         .name(this.slashCommandName)
-        .description(this.description);
+        .description(this.buildCommandDescription());
   }
 
   @Override
   protected CommandContext createContextInstance(V4Initiator initiator, V4MessageSent event) {
     return new CommandContext(initiator, event);
   }
+
+  private String buildCommandDescription() {
+    return this.requiresBotMention ? this.description + " (mention required)"
```

Comment:
I have a feeling we might come back to it in the future with users that don't want this part to appear in the description :D